cmd-record.c

```diff
@@ -87,11 +87,52 @@ static char *build_debug_domain_string(void)
 	return domain;
 }
 
+static char * get_libmcount_path(struct opts *opts)
+{
+	char *libmcount, *lib = xmalloc(4096);
+	bool must_use_multi_thread = check_libpthread(opts->exename);
+
+	if (opts->nop) {
+		libmcount = "libmcount-nop.so";
+	}
+	else if (opts->libmcount_single && !must_use_multi_thread) {
+		if (can_use_fast_libmcount(opts))
+			libmcount = "libmcount-fast-single.so";
+		else
+			libmcount = "libmcount-single.so";
+	}
+	else {
+		if (must_use_multi_thread && opts->libmcount_single)
+			pr_dbg("--libmcount-single is off because it calls pthread_create()\n");
```

Comment:
I know it's not your change but could you please change the message to `"--libmcount-single is off because it uses pthread\n"`?

Reply:
OK !